conductor/src/main.rs

```diff
@@ -414,33 +412,52 @@ async fn run(metrics: CustomMetrics) -> Result<(), Box<dyn std::error::Error>> {
                     Ok(_) => {}
                     Err(err) => {
                         error!("error restarting statefulset: {:?}", err);
+                        continue;
                     }
                 }
                 match restart_cnpg(client.clone(), &namespace, &namespace).await {
                     Ok(_) => {}
                     Err(err) => {
                         error!("error restarting cnpg: {:?}", err);
+                        continue;
                     }
                 }
-                let retry_strategy = FixedInterval::from_millis(5000).take(20);
-                let result = Retry::spawn(retry_strategy.clone(), || {
-                    get_coredb_error_without_status(client.clone(), &namespace)
-                })
-                .await;
-                if result.is_err() {
-                    error!(
-                        "error getting CoreDB status in {}: {:?}",
-                        namespace.clone(),
-                        result
-                    );
-                    metrics
-                        .conductor_errors
-                        .add(&opentelemetry::Context::current(), 1, &[]);
-                    continue;
-                }
-                let current_resource = result?;
-                let resource_json = serde_json::to_string(&current_resource).unwrap();
-                debug!("dbname: {}, current: {:?}", &namespace, resource_json);
+                let result = get_coredb_error_without_status(client.clone(), &namespace).await;
+
+                let current_resource = match result {
+                    Ok(coredb) => {
+                        // Safety: we know status exists due to get_coredb_error_without_status
+                        let status = coredb.status.as_ref().unwrap();
+                        if !status.running {
+                            // Instance is still rebooting, recheck this message later
+                            let _ = queue
+                                .set_vt::<CRUDevent>(
+                                    &control_plane_events_queue,
+                                    read_msg.msg_id,
+                                    REQUEUE_VT_SEC_SHORT,
+                                )
+                                .await?;
+                            metrics.conductor_requeues.add(
+                                &opentelemetry::Context::current(),
+                                1,
+                                &[KeyValue::new("queue_duration", "short")],
+                            );
```

Comment:
```suggestion
                            );
             continue;               
```

I forgot to `continue` here